### PR TITLE
Migrate memory_events_pane.dart to null safety.

### PR DESCRIPTION
**Pull request by polina-c**

### CODE REVIEW

File: packages/devtools_app/lib/src/screens/memory/memory_events_pane.dart [hunk 1]
```diff
@@ -112,7 +110,8 @@ class EventChartController extends ChartController {
     }
 
     if (sample.memoryEventInfo.isEventAllocationAccumulator) {
-      final allocationEvent = events.allocationAccumulator;
+      assert(events.allocationAccumulator != null);
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/screens/memory/memory_events_pane.dart:113]: remove assert

File: packages/devtools_app/lib/src/screens/memory/memory_events_pane.dart [hunk 2]
```diff
@@ -188,7 +187,7 @@ class MemoryEventsPaneState extends State<MemoryEventsPane>
 
   MemoryTimeline get _memoryTimeline => _memoryController.memoryTimeline;
 
-  ColorScheme colorScheme;
+  ColorScheme? colorScheme;
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/screens/memory/memory_events_pane.dart:190]: I think you can remove colorScheme here too

File: packages/devtools_app/lib/src/screens/memory/memory_events_pane.dart [hunk 3]
```diff
@@ -157,7 +156,7 @@ class MemoryEventsPaneState extends State<MemoryEventsPane>
   EventChartController get _chartController => widget.chartController;
 
   /// Controller for managing memory collection.
-  MemoryController _memoryController;
+  late MemoryController _memoryController;
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/screens/memory/memory_events_pane.dart:159]: use `initialize` pattern we've been using elsewhere

File: packages/devtools_app/lib/src/screens/memory/memory_events_pane.dart (hunk 1 above)
Comment (polina-c) [reply packages/devtools_app/lib/src/screens/memory/memory_events_pane.dart:113]: done

File: packages/devtools_app/lib/src/screens/memory/memory_events_pane.dart (hunk 2 above)
Comment (polina-c) [reply packages/devtools_app/lib/src/screens/memory/memory_events_pane.dart:190]: done

File: packages/devtools_app/lib/src/screens/memory/memory_events_pane.dart (hunk 3 above)
Comment (polina-c) [reply packages/devtools_app/lib/src/screens/memory/memory_events_pane.dart:159]: done

**Review by kenzieschmoll — APPROVED** [review]

**Comment by polina-c** [comment]
tested

**Pull request merged**